app/resources/css-default.template

```diff
@@ -605,6 +607,16 @@ td.submit {
   border: 2px solid #e6e6e6;
 }
 
+.query.seek .field {
```

Comment:
Should I add a comment for this? 
/* Edit field in seek query because label is too long (change line) */ ?